### PR TITLE
Update sbt to 1.10.0

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Comment by ennru** [comment]
Even the sbt plugins must be reviewed as the repo went away.

**Comment by raboof** [comment]
That's weird, I can download https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.jsuereth/pgp-library_2.12/2.1.1/ivys/ivy.xml just fine. Perhaps it was just some intermittent issue, could you retrigger the build?

**Review by ennru — APPROVED** [review]
LGTM.

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
-addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.7")
+addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.12")
```
Comment (raboof) [inline project/plugins.sbt:2]: So this was it? Weird, the error didn't seem to point at that... good find!

**Pull request merged**